package.json

```diff
@@ -128,7 +128,11 @@
     "verdaccio": "^5.0.4",
     "vite": "~4.3.9",
     "vitest": "~0.32.0",
-    "zod2md": "^0.1.1"
+    "zod2md": "^0.1.1",
+    "@nodelib/fs.walk": "^2.0.0",
+    "fast-glob": "^3.3.2",
+    "glob": "^10.3.10",
+    "globby": "^14.0.1"
```

Comment:
Could you sort these, please? A trick I use is to do something like `npm rm savbjhsfbvadsfbv` (just type anything that's not a real package name :grin:), it has the side-effect of sorting it automatically.

Reply:
done!

Reply:
BTW, could we intro `npm rm non-existin-lib` in or format task?

Reply:
There is a [Prettier plugin for `package.json`](https://github.com/matzkoh/prettier-plugin-packagejson) :bulb: